\chapter{Developmental Venous Anomalies Predispose to Malformation}
\label{chap:hht}

\blfootnote{This chapter is adapted from a study published in \italicize{bioR$\chi$iv} (Snellings et al., 2021)}
\clearpage

\section{Premise}
Cerebral cavernous malformations (CCMs) are hemorrhagic neurovascular malformations that may lead to stroke, seizures and other clinical sequelae. CCM disease may be inherited by an autosomal dominant loss of function (LOF) mutation in the genes encoding components of the CCM signaling complex: \italicize{KRIT1} \citep{couteulx1999, sahoo1999}, \italicize{CCM2} \citep{liquori2003}, or \italicize{PDCD10} \citep{bergametti2005}. Solitary sporadic CCM lesions may also occur in the absence of inherited germline mutations in CCM complex genes. Previous studies have established that CCM pathogenesis follows a genetic two-hit model where somatic mutations cause biallelic LOF in \italicize{KRIT1}, \italicize{CCM2}, or \italicize{PDCD10} to initiate lesion formation \citep{akers2009, gault2009, gault2005, mcdonald2014}. Recent studies have found that somatic mutations in \italicize{PIK3CA} and \italicize{MAP3K3} also contribute to CCM pathogenesis \citep{hong2021, ren2021, weng2021}. These discoveries opened new avenues for research and therapeutic development, but they also raised new questions about the roles of somatic mutations in familial versus sporadic lesions. 
	
The presence of multiple somatic mutations in CCMs also helps explain a long-standing mystery: the association between sporadic CCM lesions and developmental venous anomalies (DVA). DVA are the most common vascular malformation present in 6--14\% of the adult population \citep{brinjikji2017, gokce2014, jones2015, linscott2016} with the majority developing prior to the age of 20 \citep{brinjikji2017}.  When assessed by magnetic resonance imaging, an adjacent DVA is identified in 24--32\% of sporadic CCM cases \citep{abdulrauf1999, porter1997, wurm2005}, and an even greater fraction of sporadic CCMs are found to be associated with a DVA at surgery \citep{abdulrauf1999, porter1997}. One study focused on DVA reported an adjacent sporadic CCM in 6.9\% of all DVAs in a general population (116 of 1689) \citep{brinjikji2017}. These studies highlight the association between DVA and sporadic CCM. By contrast, familial CCM lesions have not been associated with DVA \citep{dammann2017}. These combined data suggest that a DVA is not required for CCM formation but may be a predisposing factor in sporadic cases. We hypothesize that DVA are caused by somatic mutations, and that the genetic origin of DVA overlaps that of CCM. In this study we probe the relationship between mutations in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, \italicize{PIK3CA}, and \italicize{MAP3K3} and explore a mechanism by which DVA act as a genetic primer for the genesis of sporadic CCM lesions. 

\section{Results}
\subsection{Mutations in \italicize{MAP3K3} and CCM Genes are Mutually Exclusive}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%				     FIGURE 1					%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[tbh]
\centering
\includegraphics[width=6in]{MAP_discovery}
\caption[\italicize{MAP3K3} Somatic Mutations and Cellular Phase in CCM]{\textbf{Mutations in \italicize{MAP3K3} are Mutually Exclusive with CCM Mutations and Occur in the Same Cells as \italicize{PIK3CA} Mutations.\\} 
\textbf{A.} Mutations present in 71 CCM samples. Disease type denotes whether the sample was familial (F), sporadic (S), or unknown (blank). The presence of somatic mutations in \italicize{PIK3CA} and \italicize{MAP3K3} are denoted by black and purple bars respectively. Germline and somatic mutations (green and blue respectively) in \italicize{KRIT1}, \italicize{CCM2}, or \italicize{PDCD10}, are shown in CCM Mut 1 with the second-hit mutation shown in CCM Mut 2 if present. \textbf{B--D.} Nuclei genotypes determined by snDNA-seq. The left and right circles in each Venn diagram shows the number of nuclei with the \italicize{PIK3CA} or \italicize{MAP3K3} mutations where the overlap shows nuclei harboring both mutations. *** P $<$ 0.0001. \textbf{E.} Summary of data presented in B--D including P values determined by comparing the observed number of double mutant nuclei to the expected value derived from a Poisson distribution as done previously \citep{ren2021}. }

\label{MAP_discovery}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

To evaluate whether sporadic and familial CCMs have distinct somatic mutation spectra we identified somatic mutations present in 71 CCMs (20 familial CCMs and 51 sporadic/presumed sporadic CCMs). Mutations in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, and \italicize{PIK3CA} were detected by targeted sequencing and/or droplet digital PCR (ddPCR) as previously described \citep{ren2021}. The common gain of function mutation in \italicize{MAP3K3} (hg38 chr17:63691212, NM\_002401.3, c.1323C$>$G; NP\_002392, p.I441M) was detected by ddPCR using a previously published probe set \citep{couto2015}. 

The p.I441M mutation in \italicize{MAP3K3} was identified in 15/51 sporadic CCMs and 0/20 familial CCMs (Figure~\ref{MAP_discovery}A). We also screened for \italicize{MAP3K3} p.I441M in 8 blood samples for which we were previously unable to identify a germline mutation in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}. None of the 8 blood samples harbored \italicize{MAP3K3} p.I441M. Notably 11/51 sporadic CCMs harbored at least 1 somatic mutation in \italicize{KRIT1}, \italicize{CCM2}, or \italicize{PDCD10}, however none of these CCMs also had a mutation in \italicize{MAP3K3} indicating that a mutual loss of the CCM complex and gain of function in MEKK3 (the protein product of \italicize{MAP3K3}) are not both required for CCM formation. As the CCM complex is known to be a direct inhibitor of MEKK3 activity \citep{zhou2015, zhou2016}, these data strongly suggest identical functional consequences of these mutations. 
	
The majority of CCM and verrucous venous malformations with a mutation in \italicize{MAP3K3} harbor the p.I441M variant \citep{couto2015, hong2021, weng2021} , however an alternative variant p.Y544H has also been identified in a venous malformation \citep{alqattan2020}. While ddPCR provides superior sensitivity and specificity compared to targeted sequencing, it is restricted to detecting a single mutation per assay. To determine whether other mutations that contribute to CCM pathogenesis---either \italicize{MAP3K3} mutations besides p.I441M, or mutations in yet undiscovered genes---we performed whole-exome sequencing (mean depth 133x) on 8 sporadic CCMs for which no somatic mutations in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, or \italicize{MAP3K3} were found. No additional mutations in \italicize{MAP3K3} were identified and no candidate variants in other genes passed QC filters (see Methods). 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%				     FIGURE 2					%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[!ht]
\centering
\includegraphics[width=5.8in]{MAP_DVA}
\caption[CCM and Adjacent DVA Harbor Identical \italicize{PIK3CA} Mutations]{\textbf{CCM and Adjacent DVA Harbor Identical Somatic Mutations in \italicize{PIK3CA}.\\}
\textbf{A--C}. Axial magnetic resonance (MR) susceptibility weighted images acquired at 3 Tesla showing CCM (circle) and associated DVA branches sampled during surgery (arrow) in individuals with CCM 5080 (A) or 5081 (B) or 5082(C). The inset red box in C shows the region expanded to the right with the CCM and DVA marked. \textbf{D--E}. Somatic mutations in \italicize{PIK3CA} (D) and \italicize{MAP3K3} (E) in CCM (top panels) and the associated DVA (bottom panels) from samples 5081, 5082, and 5083. Mutations were detected by droplet digital PCR (ddPCR) and shown as the fluorescence of the reference probe on the x-axis, and the mutant probe on the y-axis. Droplets containing the reference allele, mutant allele, both, or neither, are colored in green, blue, orange, and black respectively. Percentage inset into each graph shows the variant allele frequency for the displayed mutation. If the mutation was determined to be present, the percentage is blue, else the percentage is red.}

\label{MAP_DVA}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Cellular Phase of Somatic \italicize{MAP3K3} and \italicize{PIK3CA} Mutations}

While somatic mutations in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, and \italicize{MAP3K3} are mutually exclusive, somatic gain of function mutations in \italicize{PIK3CA} may co-occur with any other mutation (Figure~\ref{MAP_discovery}A). We have previously shown that co-occurring mutations in \italicize{KRIT1}/\italicize{CCM2} and \italicize{PIK3CA} occur in the same clonal population of cells \citep{ren2021}. To determine whether \italicize{MAP3K3} and \italicize{PIK3CA} mutations co-exist in the same cells we performed single-nucleus DNA-sequencing (snDNA-seq) on frozen tissue from three surgically resected CCMs determined to harbor both mutations (Figure~\ref{MAP_discovery}B--D). 

In CCMs 5002 and 5030, the vast majority of mutant nuclei harbor both mutations in \italicize{MAP3K3} and in \italicize{PIK3CA} indicating that these mutations co-exist in the same cells. In CCM 5032, 37\% (19/51) of mutant nuclei harbor both mutations. While this is a far lower fraction compared to other samples, it is significantly higher than may be expected by chance when sampling from 1405 total nuclei (\italicize{P} = $1.3\times10^{-05}$, Figure~\ref{MAP_discovery}E). In bulk genetic analysis, the allele frequencies of \italicize{PIK3CA} and \italicize{MAP3K3} mutations detected in CCM 5002 were 19\% and 13\% respectively. In snDNA-seq the allele frequencies of these mutations increased to 54\% and 55\% respectively. This difference likely reflects the mosaic nature of CCMs. As snDNA-seq requires nuclei harvested from frozen tissue, we must sample a new area of the frozen lesion than was sampled for bulk sequencing. Sampling from different sites of the same lesion often results in minor changes in allele frequency, however the drastic change in allele frequency we find in CCM 5002 suggests either that our initial sample of the lesion for bulk sequencing contained largely non-lesion tissue, or an uneven distribution of mutant cells in the lesion. 

\subsection{CCMs and DVA Harbor a Shared Mutation in \italicize{PIK3CA} }

Many sporadic CCMs are found in the vicinity of a developmental venous anomaly (DVA). We hypothesized that CCM and DVA may have a common genetic origin, specifically that DVA may be a genetic precursor to CCM. To determine whether DVA and CCMs originate from a shared mutation, we collected three sporadic CCMs and sampled a portion of the associated DVA obtained during surgery (Figure~\ref{MAP_DVA}A--C). Assays for mutations via ddPCR revealed that all three CCMs have a somatic activating mutation in \italicize{PIK3CA} and that the same mutation is present within the paired DVA samples at lower frequency (Figure~\ref{MAP_DVA}D). Furthermore, ddPCR revealed that two of the CCMs harbored a mutation in \italicize{MAP3K3} in addition to the previously noted mutation in \italicize{PIK3CA}. However, unlike the \italicize{PIK3CA} mutation, the \italicize{MAP3K3} mutation was entirely absent from both DVA samples (Figure~\ref{MAP_DVA}E). The presence of the \italicize{PIK3CA}, but not the \italicize{MAP3K3}, mutation in the DVA confirms that the \italicize{PIK3CA} mutation in the DVA did not arise via cross-sample contamination. The presence of multiple somatic mutations in these CCMs allows us to infer the developmental history of the lesion. The cancer field commonly uses the presence or absence of somatic mutations in clonal populations to track the evolutionary history of a tumor \citep{jiao2014, loo2018}. Recent studies have expanded on this approach to use somatic mutations as endogenous barcodes to track embryonic development \citep{bizzotto2021}. Using this same approach, we infer that the DVA was the first lesion to develop and that the associated CCM is derived from cells of the DVA following a somatic mutation in \italicize{MAP3K3}. 

\subsection{Plasma miRNAs Reflect PI3K Activation in DVA Patients}

In addition to assaying the presence of \italicize{PIK3CA} mutations in DVA associated with CCM, we would ideally also assay DVA that are not associated with CCM. Unfortunately, DVA are benign malformations and are not resected unless associated with an additional pathology. This has precluded the direct assessment of \italicize{PIK3CA} mutations in DVA without a CCM.   To address this limitation, we sought another source of tissue that could be assayed for indirect evidence of \italicize{PIK3CA} activation.  Thus, we collected plasma from individuals with DVA without a CCM and measured circulating miRNAs that might serve as biomarkers reflecting \italicize{PIK3CA} activity \citep{mori2019}.

We sequenced the plasma miRNomes of 12 individuals with a sporadic CCM associated with a DVA (CCM + DVA), 6 individuals with a DVA without a CCM (DVA only), and 7 healthy controls. Three plasma miRNAs were DE in the DVA only group when compared to healthy controls (P $<$ 0.05; false discovery rate (FDR) corrected). Two of these DE miRNAs, \italicize{miR-134-5p} (FC = 0.10) and \italicize{miR-92a-3p} (FC = 3.10), putatively target \italicize{PIK3CA} and \italicize{PIK3CB} respectively, which are both components of the PI3K/AKT/mTOR pathway (Figure~\ref{MAP_miRNA}).

In addition, 18 plasma miRNAs were DE in CCM + DVA when compared to DVA only (P $<$ 0.05; FDR corrected). Two of the 18 DE miRNAs, \italicize{miR-122-5p} (FC = 5.25) and \italicize{miR-182-5p} (FC = 0.21), target \italicize{AKT3} and \italicize{MAP3K3} respectively, linking them to both the PI3K/AKT/mTOR and MAPK/ERK pathways. Additionally, \italicize{let-7c-5p} (FC = 12.64) targets both \italicize{PIK3CA} and \italicize{MAP3K3}, consistent with the role of these two pathways in sporadic CCM pathogenesis (Figure~\ref{MAP_miRNA}). Of interest, \italicize{let-7c-5p} also targets \italicize{COL1A1}, a DEG within the transcriptome of human sporadic CCM lesions (see Supplementary Data). This gene is associated with PI3K/AKT signaling, platelet activation, and ECM-receptor interaction KEGG pathways \citep{kanehisa2021, kanehisa2000}, all of which have previously been implicated in CCM disease \citep{faurobert2013, hong2021, ramirez2019, ren2021}.

Additionally, 28 DE plasma miRNAs were identified between CCM + DVA and healthy controls (P $<$ 0.05; FDR corrected). Four of these miRNAs putatively target \italicize{PIK3CA}: \italicize{miR-148a-3p} (FC = 3.27), \italicize{miR-148b-3p} (FC = 2.64), \italicize{miR-128-3p} (FC = 2.55) and \italicize{let-7c-5p} (FC = 4.20), which also targets \italicize{MAP3K3}. 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%				     FIGURE 4					%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[t!]
\centering
\includegraphics[width=6in]{MAP_miRNA}
\caption[Differentially Expressed micro-RNAs in CCM and DVA]{\textbf{Differentially Expressed micro-RNAs in CCM and DVA. \\} 
Two miRNAs differentially expressed (DE) in the plasma of individuals with DVA without a CCM, \italicize{miR-134-5p} and \italicize{miR-92a-3p}, putatively target genes in the Kyoto Encyclopedia of Genes and Genomes (KEGG) PI3K/AKT signaling pathway. In the plasma of individuals with sporadic CCMs with an associated DVA, \italicize{miR-122-5p}, \italicize{miR-182-5p}, and \italicize{let-7c-5p} were DE and found to target genes within both PI3K/AKT and MAPK signaling KEGG pathways. All results were P $<$ 0.05; false discovery rate corrected. }

\label{MAP_miRNA}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\subsection{Relative Probability of Multiple Somatic Mutations }

The association between DVA and sporadic---but not familial---CCMs suggests that there are at least two genetic trajectories by which a CCM may develop. The first trajectory is via a quiescent CCM caused by an initial mutation in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, or \italicize{MAP3K3}. The second trajectory is via a DVA caused by an initial mutation in \italicize{PIK3CA} with subsequent mutations in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, or \italicize{MAP3K3} leading to CCM formation. To understand whether one trajectory is favored in familial vs sporadic CCMs we use a simplified model to estimate the relative probability of CCM complex LOF (\italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}), \italicize{MAP3K3} GOF, and \italicize{PIK3CA} GOF. 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%				     FIGURE 3					%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{figure}[h!]
\centering
\includegraphics[scale=.19]{MAP_model}
\caption[Genetic Model of CCM Pathogenesis]{\textbf{Genetic Model of CCM Pathogenesis. \\} 
The genetic trajectories that underly familial and sporadic CCM pathogenesis. The initial path of familial and sporadic CCMs are denoted by arrow colors (green and orange respectively) with the relative probability of each path denoted by the size of the arrow. Familial CCM preferentially develop via a LOF mutation in the CCM complex (\italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}) to develop a quiescent CCM that may acquire an additional mutation in \italicize{PIK3CA} that drives lesion growth. Sporadic CCM are more likely to develop via a DVA caused by a \italicize{PIK3CA} mutation creating a clonal population of cells that may subsequently acquire a CCM complex LOF or \italicize{MAP3K3} GOF. }

\label{MAP_model}
\end{figure}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Somatic mutagenesis is a complex process that depends on many variables including cell turnover rates, age, exposure to mutagens, and numerous genetic factors which facilitate DNA synthesis and repair. As a result, estimating absolute mutation rate is not feasible. However, by assuming a uniform mutation rate across these genes, we can determine the relative rate of CCM complex LOF, \italicize{MAP3K3} GOF, and \italicize{PIK3CA} GOF as simply the number of mutations that result in LOF or GOF. Determining this value for \italicize{MAP3K3} and \italicize{PIK3CA} is straightforward. To date there is only a single mutation in \italicize{MAP3K3} (p.I441M) that has been reported in CCMs. The spectrum of mutations in \italicize{PIK3CA} has been well documented in the catalogue of somatic mutations in cancer (COSMIC). In the absence of functional assays for each mutation, we define a GOF mutation as any mutation that accounts for $>$1\% of all \italicize{PIK3CA} mutations COSMIC which was determined to be 10 mutations. The number of CCM complex LOF mutations was determined by considering all possible single nucleotide variants that would result in a premature stop codon or disrupt a canonical splice site (see Methods). The average of these values for all three genes is 430 mutations (\italicize{KRIT1} = 718, \italicize{CCM2} = 353, \italicize{PDCD10} = 220). This is a very conservative estimate and should be considered a lower bound, since the majority of CCM complex LOF mutations are the result of frameshift mutations which are not accounted for here. 

From the relative probability of each mutation, it is clear that familial CCMs will almost always develop via CCM complex LOF (Figure~\ref{MAP_model} top), consistent with the lack of association between familial CCM and DVA. The probability of two CCM complex LOF mutations in trans and in the same cell (as required for CCM complex LOF in sporadic CCMs) is far lower than for a single mutation (as required for CCM complex LOF in familial CCMs). In this study we identified 6 sporadic CCMs with biallelic somatic mutations in a CCM complex gene, therefore probability of this event is likely of similar magnitude to \italicize{MAP3K3} GOF which we identified in 15 sporadic CCMs. Assuming the same relative probability for CCM complex LOF as \italicize{MAP3K3} GOF in sporadic CCMs, we determine that sporadic CCM are at least 5x more likely to develop via \italicize{PIK3CA} GOF than CCM complex LOF or \italicize{MAP3K3} GOF (Figure~\ref{MAP_model} bottom). This finding is consistent with the strong association between DVA and sporadic CCM.



\section{Discussion}

\subsection{\italicize{MAP3K3} GOF and CCM LOF Have Similar Molecular Effects}
In this study we have further interrogated the relationship between somatic mutations in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, \italicize{MAP3K3}, and \italicize{PIK3CA} which contribute to the pathogenesis of CCM. We find that somatic mutations in \italicize{MAP3K3} are not present in CCMs from individuals with familial CCM, consistent with a recent study \citep{weng2021}.  We find that sporadic CCMs may harbor mutations in \italicize{MAP3K3}, \italicize{KRIT1}, \italicize{CCM2}, or \italicize{PDCD10}, but that the lesion will only have mutations in one of these genes. This implies that mutations in any of \italicize{MAP3K3}, \italicize{KRIT1}, \italicize{CCM2}, or \italicize{PDCD10} are sufficient for CCM formation, without the need for mutations in a second gene. As \italicize{KRIT1}, \italicize{CCM2}, and \italicize{PDCD10} are all members of the CCM signaling complex, this reflects the fact that the loss of any component of the complex prevents normal signal transduction. Similarly, as the CCM complex is a direct inhibitor of \italicize{MAP3K3} activity \citep{zhou2015}, this pathway may be activated by either CCM complex LOF or by \italicize{MAP3K3} GOF, but the mutual exclusivity of mutations in these genes suggests that only one of these events is necessary for lesion formation. 

\subsection{DVA Clonal Expansion Promotes Additional Mutations}
CCMs often develop as the result of multiple somatic mutations that co-exist within the same cells as we show with snDNA-seq. Although several somatic mutations occur in every cell division \citep{rodin2021}, the specificity of the mutations in CCM translates to a very low chance of acquiring these mutations within a single cell. This is especially true of somatic mutations in \italicize{MAP3K3} and \italicize{PIK3CA}, both of which have very narrow spectra of activating mutations. Despite this improbability, the accumulation of these mutations in CCM seems to occur frequently. One possible explanation for this phenomenon is that CCMs have an increased rate of somatic mutations. There is little evidence supporting this theory and such a mechanism is difficult to conceive in cases of sporadic CCM where individuals have no known genetic predisposition to CCM. An alternative explanation is that after an initial somatic mutation, the singly-mutated cell undergoes clonal expansion to form an intermediate lesion. In this study we identify 7 CCMs with either biallelic LOF in a CCM complex gene or \italicize{MAP3K3} GOF in the absence a \italicize{PIK3CA} mutation, suggesting that \italicize{PIK3CA} activation is not required for CCM formation. Furthermore, previous work in mouse models has shown that loss of a CCM complex gene (with WT \italicize{Pik3ca}) leads to clonal expansion of the mutant cells and in later stages of growth, incorporation of WT endothelial cells \citep{detter2018, malinverno2019}. As a result of this clonal expansion, the probability of creating a double-mutant cell increases by a factor of the clonal population size as there are more cells in which the second mutation may occur. Somatic mutations in this clonal population may be acquired by genome replication during expansion or after expansion as mutations accumulate via DNA damage and error-prone repair as has been reported in other quiescent cell types \citep{lodato2018}. 

The data presented in this study suggest that DVA are an intermediate lesion as posited by the latter explanation. Genetic analysis of DVA and paired CCM show that at least some DVA develop following a somatic activating mutation in \italicize{PIK3CA}. Furthermore, plasma miRNA analysis of individuals with DVA-associated CCM revealed differentially expressed miRNAs that putatively target \italicize{PIK3CA} and \italicize{MAP3K3}, the same genes that are mutated in these lesions. By contrast, individuals with DVA but no CCM differentially expressed miRNAs that putatively target \italicize{PIK3CA}, but not \italicize{MAP3K3}. These data are consistent with the distribution of mutations we observe in CCM and DVA, and the differential regulation of these miRNAs may reflect an attempt to compensate for dysregulation of PI3K and MAPK signaling due to the somatic mutations. 

The presence of \italicize{PIK3CA} mutations in DVA suggests that DVA act as a genetic precursor to CCM, which would account for the strong association between sporadic CCM and DVA. Likewise, DVA are not associated with familial CCM because the presence of an inherited germline mutation in a CCM gene strongly biases probability towards a CCM gene somatic mutation occurring first, as there exist many different mutations that may cause LOF, but far fewer that would cause GOF in \italicize{PIK3CA}. 

This study is the first to collect CCM and associated DVA for genetic analysis. Collecting tissue from CCM-associated DVA is challenging; however, collecting tissue from DVA not associated with CCM is yet more challenging as DVA are considered benign and are therefore not resected. We have attempted to address this limitation by studying biomarkers of PI3K activity which can be assayed noninvasively in blood plasma. Assaying the presence of \italicize{PIK3CA} mutations in DVA not associated with CCM will be the domain of future studies, but the data we present here demonstrate a clear link between DVA and \italicize{PIK3CA}, and suggest a model that explains the long recognized\textemdash but poorly understood\textemdash association between CCM and DVA. 

\subsection{DVA May Contribute to Additional Phenotypes}
While we are unable to address the presence of \italicize{PIK3CA} mutations in DVA not associated with CCM, it is worth noting that DVAs have been associated with other PI3K-related disorders\citep{brinjikji2018, dhamija2018, rasalkar2010, roux2020, santucci2008, tan2007, ucler2019} including some cancers and neurological malformations, suggesting that DVA may have a role, possibly even as a genetic primer, in these other diseases. 



\section{Methods}
\subsection{Sample Collection}
	Surgically resected CCMs were obtained from the University of Chicago, the Barrow Neurological Institute, and the Angioma Alliance biobank. Additional DVA tissue was discretely dissected from the lesion during surgical resection of the associated CCM at the University of Chicago. This study was approved by each institution’s respective Institutional Review Board. 

\subsection{DNA Extraction}
	DNA from CCM and DVA samples was extracted using the DNeasy blood and tissue kit (QIAGEN, catalog number 69504) per the manufacturers protocol. DNA purity was determined by Nanodrop and concentration was determined using the Qubit dsDNA BR assay kit (Invitrogen, catalog number Q32850) per the manufacturers protocol. 

\subsection{Droplet Digital PCR}
Detection of \italicize{MAP3K3} p.I441M was performed via ddPCR using a previously described probe set\citep{couto2015}. Assays were performed using 30--100ng of DNA with the QX200 AutoDG system (BioRad) and quantified with the QX200 droplet reader (BioRad). Analysis was performed with the QuantaSoft software (BioRad). 

\subsection{Sequencing}
	A total of 8 sporadic CCMs with no identified mutation in \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, or \italicize{MAP3K3} (5001, 5005, 5006, 5022, 5024, 5036, 5078, and 5081) were used for whole-exome sequencing prepared using the SureSelect Human All Exon V7 probe set (Agilent, Design ID S31285117) per the manufacturers protocol. Prepared libraries were sequenced on one lane of a NovaSeq 6000 S4 flow cell for a mean depth of 133x. 

\subsection{Sequence Analysis}
	Sequencing data was processed using the Gene Analysis Toolkit (GATK, Broad Institute) while following the GATK best practices for somatic short variant discovery using Mutect2. Secondary variant detection was performed using gonomics (https://github.com/vertgenlab/gonomics) and bcftools mpileup to manually examine \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, and \italicize{MAP3K3} for somatic variants. Putative variants were annotated using Funcotator (GATK), the catalog of somatic mutation in cancer (COSMIC), and the genome aggregation database (gnomAD). Putative variants were filtered according to the following criteria: greater than 50x total coverage, less than 90\% strand specificity, greater than 5 reads supporting the alternate allele, greater than 1\% alternate allele frequency, less than 1\% population allele frequency, and predicted protein/splicing change. 

\subsection{Single-Nucleus DNA Sequencing}
Nuclei isolation, snDNA-seq, and analysis were performed as previously described\citep{ren2021}. 

\subsection{miRNA Extraction and Sequencing}
Total plasma RNA was extracted from the plasma of 12 individuals with a sporadic CCM and an associated DVA (CCM + DVA), 6 individuals with DVA and without a CCM (DVA only), and 7 healthy controls using the miRNeasy Serum/Plasma Kit (Qiagen, Hilden, Germany) following the manufacturer isolation protocol. Diagnosis of CCM with an associated DVA, as well as DVA without a CCM lesion was confirmed on susceptibility weighted MR imaging. Illumina small RNA-Seq kits (Clontech, Mountain View, CA, USA) were then used to generate cDNA libraries, and sequencing was completed with the Illumina HiSeq 4000 platform (Illumina, San Diego, CA, USA), with single-end 50bp reads, at the University of Chicago Genomics Core. Differential miRNA analyses were completed between (1) CCM + DVA to DVA only and then (2) DVA only to healthy controls. The differentially expressed miRNAs were identified having P $<$ 0.05, FDR-corrected. All analyses were completed using the sRNAToolbox and DESeq2 R packages\citep{love2014, rueda2015}.

\subsection{Identification of Putative Targets}
miRWalk 3.0 was used to identify the putative gene targets of each of the DE miRNAs, using a random forest tree algorithm with a bonding prediction probability higher than 95\% on the 3 different gene locations (3' UTR, 5' UTR, and CDS)\citep{sticht2018}. Putative gene targets of the DE miRNAs were identified in at least 2 of the 3 databases. DE miRNAs between (1) CCM + DVA and DVA only as well as (2) DVA only and healthy controls were then analyzed for potential targeting of the PI3K, AKT, and MAPK gene families. 

\subsection{Estimating Possible LOF Mutations in \italicize{KRIT1}, \italicize{CCM2}, and \italicize{PDCD10}}
The majority of LOF mutations in \italicize{KRIT1}, \italicize{CCM2}, and \italicize{PDCD10} result in either: creation of a premature stop codon (nonsense); disruption of a canonical splice site; or a frameshift. The first two of these are typically caused by a single nucleotide variant (SNV) and can be determined from the gene sequence. However, frameshift variants resulting from insertions and deletions may occur in any exon regardless of sequence context. This prevents a meaningful comparison between indel and SNV events, without making assumptions about the rate of somatic SNV and indel events. To determine a conservative lower bound for the number of LOF mutations in the CCM complex genes, we only consider nonsense and splice site mutations. The number of nonsense and splice site mutations were determined from the sequences of \italicize{KRIT1} (ENST00000394507.5), \italicize{CCM2} (ENST00000258781.11), and \italicize{PDCD10} (ENST00000392750.7). Nonsense mutations were considered any SNV in the coding sequence that would result in an in frame stop codon prior to the stop codon in the reference sequence. Splice site mutations were considered any SNV in the nearly invariant sequences at each exon-intron boundary that mediate canonical splicing. 
	
\subsection{Sporadic CCM Transcriptome}
Laser micro-dissected neurovascular units (NVUs) from sporadic CCM lesions were sequenced for transcriptomic analyses in comparison to micro-dissected NVUs from human brain microvasculature. The differential analyses identified 426 DE genes (DEGs) (P $<$ 0.05; FDR corrected; FC $>$ $\vert$1.5$\vert$). Additionally, 8 pathways from the Kyoto Encyclopedia of Genes and Genomes (KEGG) were enriched (P $<$ 0.05, FDR corrected).  


\subsection{Sporadic Transcriptome and KEGG Pathway Analyses}

Six human sporadic CCM lesions were surgically resected prior to embedding in optimal cutting temperature, snap-freezing, and storing at -80$^{\circ}$C. Control brain tissue was collected during autopsy from three subjects lacking neurological disease, fixed in formalin, and embedded in paraffin blocks. Five-\textmu m tissue sections were mounted on Leica glass slides (Leica Biosystems Inc) and were stained, in accordance with the manufacturer’s protocols, with HistoGene (Applied Biosystems) for frozen tissue and Paradise stain (Applied Biosystems) for paraffin-embedded tissue. The neurovascular units (NVUs) from sporadic CCM lesions and normal brain capillaries were then collected using laser capture microdissection and stored at -80$^{\circ}$C. RNA was isolated using an RNA extraction kit (RNeasy Micro Kit, Qiagen). cDNA libraries were then generated with low-input strand-specific RNA-Seq kits (Clontech) and sequenced using the Illumina HiSeq 4000 platform with single-end 50bp reads. Differentially expressed genes were defined as P $<$ 0.05, FDR corrected. 

Enriched Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways\citep{kanehisa2021, kanehisa2000} were obtained for the DE genes (DEGs) in the sporadic transcriptome with fold change values greater than 1.5 (P $<$ 0.05, FDR corrected). The sporadic transcriptome was compared to the putative targets of the DE miRNAs between (1) CCM + DVA and DVA only as well as (2) DVA only and healthy controls to obtain a set of overlapping genes. Enriched Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways were obtained for the overlapping genes, using a database and knowledge extraction engine with a Bayes factor greater than 3. 

\section{Contributions and Acknowledgements}
This chapter is adapted from a study published in \italicize{bioR$\chi$iv} (Snellings et al., 2021) with the following authors: Daniel A. Snellings, Romuald Girard, Rhonda Lightle, Abhinav Srinath, Sharbel Romanos, Ying Li, Chang Chen, Aileen A. Ren, Mark L. Kahn, Issam A. Awad, Douglas A. Marchuk. DAS designed and performed the genetic studies of human CCM lesions and wrote the manuscript. IAA performed surgical resection of CCM and DVA samples used in this study. RG, RL, AS, SR, YL, CC and IAA performed plasma miRNA sequencing and analysis. AAR and MLK assisted with experimental design. RG, IAA, and DAM designed experiments and wrote the manuscript.

We thank the patients who donated tissue for this study. We thank Angioma Alliance, the Barrow Neurological Institute, and the University of Chicago for patient enrollment and sample collection. Nucleus sorting was performed in the Duke Human Vaccine Institute Research Flow Cytometry Shared Resource Facility. We thank Duke University School of Medicine for use of the Sequencing and Genomic Technologies Shared Resource for library preparation and sequencing. These studies were supported by National Institute of Health grants, P01NS092521 (DM, IA, MK), F31HL152738 (DS).
