\chapter{\italicize{PIK3CA} Mutations Fuel Cerebral Cavernous Malformation Growth}
\label{chap:pik3ca}

\blfootnote{This chapter is adapted from a study published in \italicize{Nature} (Ren \& Snellings et al., 2021)}
\clearpage

\section{Introduction}
Vascular malformations such as cerebral cavernous malformations (CCMs) that arise in the central nervous system are an important cause of stroke and disability in younger individuals \citep{heiskanen1993, fischer2013}.  Most CCMs arise sporadically as single lesions, but a minority present as part of a familial, autosomal dominant form of the disease that is associated with multiple lesions \citep{cavalcanti2012}. Classic genetic studies have associated familial CCM disease with heterozygous germline loss of function mutations in three genes, \italicize{KRIT1}, \italicize{CCM2}, and \italicize{PDCD10}, that encode the components of a heterotrimeric protein complex (the “CCM complex”) \citep{fisher2014, plummer2005}. Subsequent studies have demonstrated that CCM lesions harbor an additional somatic mutation in the same gene as the germline mutation, implicating biallelic loss of function of the affected CCM gene as the cause of the disease \citep{gault2005, akers2009}. Consistent with this monogenic loss of function mechanism, sporadic CCMs harbor biallelic somatic mutations in one of the CCM genes, resulting in homozygous loss of function \citep{mcdonald2014}. Mouse models confirm that deletion of any of the CCM genes in the brain endothelial cells of neonatal mice confers CCM lesions, proving a causal role for loss of CCM complex function in this disease.  These studies have generated the current genetic model of CCM pathogenesis in which biallelic loss of function mutations in a single CCM gene is sufficient for CCM lesion development. 

Serial imaging studies to define the natural history of human CCMs has revealed that most are slow-growing and clinically silent \citep{akers2017, alshanisalman2012, horne2016}. In contrast, those that cause stroke and seizure are typically fast-growing and associated with repeated lesional hemorrhage \citep{awad2019, porter1997}. Such aggressive, symptomatic lesions are surgically resected if possible to prevent or treat associated neurologic complications, but surgery is associated with high morbidity and cost and is impractical for patients with multiple lesions or lesions in less reachable locations such as the spinal cord. Why a subset of CCM lesions exhibits rapid growth associated with clinical symptoms is unknown. Recent mouse and human studies suggest that a gut microbiome containing more invasive gram negative bacteria or an impairment of the gut barrier that blocks translocation of bacterial products such as lipopolysaccharide may modulate CCM growth through effects on TLR4-MEKK3-KLF2/4 signaling in brain endothelial cells \citep{tang2019, tang2017, polster2020}. Plasma biomarkers of angiogenesis have also been correlated with lesional clinical activity \citep{girard2018, lyne2019}. However, individuals with familial CCM disease who harbor numerous silent lesions identified by MRI imaging also often manifest symptomatic hemorrhage and aggressive growth of a single lesion \citep{polster2019}. Thus the current understanding of the environmental and genetic factors that contribute to CCM growth fails to explain important aspects of the disease natural history, especially the emergence of rapidly growing symptomatic lesions which account for the majority of clinically significant outcomes.

Recent studies of sporadic vascular malformations have identified acquired gain of function mutations in a number of central signaling pathways, including the RAS/MAPK/ERK pathway in congenital hemangiomas and capillary malformations and the PI3K/AKT/mTOR pathway in venous and lymphatic malformations \citep{tenbroek2019, rodriguezlaguna2019, castillo2019, wetzelstrong2017, luks2015, limaye2015} (and reviewed in \citep{queisser2018}). Many of these gain of function mutations, e.g. those in \italicize{PIK3CA}, the catalytic subunit of PI3K, are identical to those that have been identified in cancer cells \citep{dastillo2016, castel2016, limaye2015, koren2015, samuels2005}. However, unlike cancer, in which mutations in multiple driver genes such as tumor suppressor genes and oncogenes combine to promote growth \citep{bailey2018, mcgranahan2015}, the pathogenesis of vascular malformations has been considered monogenic. The studies described below reveal that symptomatic CCM disease arises through a cancer-like paradigm in which the accumulation of multiple somatic mutations in the same cell results in both the loss of a vascular malformation suppressor gene (i.e. the CCM gene) and the gain of vascular malformation growth gene (i.e. \italicize{PIK3CA}). These studies reveal that clinically significant cavernous malformations arise through a compound genetic mechanism like that previously described for cancer, identify PI3K signaling as a major downstream effector pathway in CCM disease, and suggest that symptomatic CCMs may be effectively treated with the approved drug rapamycin (aka Sirolimus). 


\section{Results}
\subsection{\italicize{PIK3CA} Mutations Occur in Familial and Sporadic CCMs}
To determine whether human CCM lesions harbor gain of function mutations in \italicize{PIK3CA} or other genes that have been associated with increased cell growth and proliferation, 79 surgically resected CCM lesions (a single lesions per individual) were sequenced with a targeted panel of 66 genes, including the three causal CCM genes, genes involved in PI3K signaling and associated pathways, other oncogenic pathway genes, and other genes found to be mutated in vascular malformations (full gene list in methods). The collected CCM lesions were classified as “familial”, “sporadic” or “unknown” based on genetic and clinical evaluations (described in the methods). To ensure that any sequence variants identified in the CCM lesions were specific to CCM disease, 68 distinct surgically resected human brain arteriovenous malformations (bAVMs) were collected and sequenced. Like CCM lesions, bAVMs are neurovascular malformations enriched in vascular endothelial cells; thus the cellular composition of bAVMs is similar to that of CCM lesions. Since bAVMs and CCMs share a similar biological organization but arise due to distinct pathogenic mechanisms bAVMs provide a control with which to identify mutations in CCM lesions that are specific to CCM pathogenesis. 

Variants called from the sequencing data were filtered to select for those with at least 5 supporting alternate reads, a variant allele frequency greater than 0.5\%, predicted functional consequence, and several other filtering criteria. Remarkably, sequencing revealed that 56/79 (71\%, \italicize{P}=1.23 $\times$ $10^{-12}$) resected human CCM lesions harbor a somatic mutation in \italicize{PIK3CA} (Fig. 4a). By contrast, none of the 68 bAVM samples harbored a somatic mutation in \italicize{PIK3CA}. The variant allele frequency of the \italicize{PIK3CA} mutations in CCM lesions ranged between 0.7\% and 17.5\% with a mean of 4.7\%, suggesting mosaicism within the CCM lesion. All of the \italicize{PIK3CA} mutations occurred at known hotspots in the catalogue of somatic mutations in cancer (COSMIC and Fig. 4d). Significantly, analysis of 62 other genes with listings in the COSMIC database failed to identify mutations in any genes other than \italicize{PIK3CA} and the CCM genes. No mutations were found in other components of the PI3K pathway, including \italicize{PTEN} and \italicize{AKT1/2/3}, revealing strong specificity for \italicize{PIK3CA} mutations in CCM. The three most common \italicize{PIK3CA} mutations identified in CCM lesions (E542K, E545K, and H1047R) were validated with droplet digital PCR and SNaPshot (single nucleotide extension) assays. Mutations in \italicize{PIK3CA} were detected in 14/21 known familial lesions (9/15 \italicize{KRIT1}, 4/5 \italicize{CCM2}, 1/1 \italicize{PDCD10}), and 12/15 known sporadic lesions (Fig. 4b). Each CCM lesion harbored no more than one somatic mutation in \italicize{PIK3CA}, and all of the \italicize{PIK3CA} mutations identified in CCM lesions have previously been determined to activate PI3K signaling \citep{dogruluk2015}.  

\subsection{CCMs Harbor Multiple Somatic Mutations in Different Genes}
Previous studies have demonstrated that human CCM lesions harbor somatic mutations in one of the three causal CCM genes. In this cohort, somatic mutations in CCM genes were identified in 24/79 (30\%) of CCM lesions. The relatively low discovery rate of somatic CCM mutations may reflect types of mutations that are not detectable with short-read sequencing, such as large indels or chromosomal rearrangements. Notably, in the CCM lesions in which we positively identified a somatic loss of function CCM mutation, 21/24 (88\%) also harbored a somatic gain of function \italicize{PIK3CA} mutation. This apparent enrichment in the co-detection of CCM and \italicize{PIK3CA} somatic mutations is consistent with poor sample quality that reduced sensitivity and/or low variant allele frequency in many lesions. Thus the true frequency of \italicize{PIK3CA} mutations in CCM lesions is likely to be higher than the 71\% reported above. The identification of multiple mutations in CCM genes is consistent with the previously described two-hit model of CCM pathogenesis with the addition of a third hit in \italicize{PIK3CA}.  In 9 samples of familial CCM (\ddag ~ on Fig. 4a), we detected distinct loss of function germline and somatic mutations in the same CCM gene in addition to a gain of function mutation in \italicize{PIK3CA} for a total of 3 genetic hits. In 6 samples of presumed sporadic CCM (*  on Fig. 4a), we detected two distinct loss of function somatic mutations in the same CCM gene in addition to a gain of function mutation in \italicize{PIK3CA} for a total of 3 genetic hits. These data indicate that no fewer than three independent somatic mutation events contributed to the pathogenesis of those lesions. A comparison of variant allele frequency between CCM and \italicize{PIK3CA} somatic mutations in lesions where both mutations were found revealed no significant correlation (P $>$ 0.15) (Fig. 4c). This balanced mutation frequency is consistent with a pathogenic mechanism in which both CCM and \italicize{PIK3CA} somatic mutations arise in a single cell during lesion formation, or one in which CCM mutant cells and \italicize{PIK3CA} mutant cells co-exist in similar numbers. Finally, the specific mutant \italicize{PIK3CA} alleles identified in resected human CCM lesions closely mirrored those identified in human cancers in the COSMIC database (Fig. 4d), consistent with a shared molecular and cellular mechanism. 

\subsection{\italicize{PIK3CA} and CCM/\italicize{MAP3K3} Mutations in the Same Cell}
The identification of both \italicize{PIK3CA} and CCM gene somatic mutations in CCMs raises a critical question: Do these mutations occur in the same cell, or are these mutations in two distinct clonal populations that intermix to form a CCM. To address this question we performed single-nucleus DNA sequencing (snDNA-seq) on 3 sporadic and 2 familial CCMs using the Tapestri platform \citep{xu2019} (REF Xu et al 2019, PMID: 31366893). Nuclei isolated from frozen tissue were stained with DAPI and subjected to fluorescence-activated nucleus sorting to isolate single nuclei for input into the Tapestri instrument, where the nuclei were partitioned into droplets and the exons of \italicize{KRIT1}, \italicize{CCM2}, \italicize{PDCD10}, and \italicize{PIK3CA} were amplified (Fig. 4e). Bulk sequencing of sample 5003 identified one somatic mutation in \italicize{PIK3CA} and two somatic mutations in \italicize{KRIT1}. These same somatic mutations were identified in the snDNA-seq data which show that the majority of somatic mutant nuclei harbored all three mutations. In sporadic CCMs 5038 and 5079 only one somatic CCM gene mutation was called in addition to the \italicize{PIK3CA} somatic mutation. In sample 5079 the second somatic CCM gene mutation is clearly present in total reads, however due to poor efficiency of the amplicon there were insufficient reads per nuclei to reliably establish cellular phase with the other two mutations. Data from both 5038 and 5079 show that the majority of mutant cells harbor both the somatic \italicize{PIK3CA} and CCM gene mutations. Likewise snDNA-seq data for familial CCMs 5065 and 5073 show the majority of somatic mutant nuclei (excluding nuclei with only the germline CCM gene mutation) harbor the \italicize{PIK3CA} mutation and both CCM gene mutations (Fig 4f, Table 1). While the majority of somatic mutant nuclei in all 5 samples harbor all of the identified somatic mutations, notably there is a smaller number of nuclei observed with each possible combination of genotypes. This observation is highly unlikely to reflect genuine biology as the creation of all genotypes would require identical somatic mutations to occur in multiple clonal populations within the CCM. Some of the observed genotype combinations may represent intermediate clonal populations that formed prior to acquiring the full set of mutations, however the majority of these genotype combinations are likely due to allelic dropout (ADO)---a common technical artifact in single-nucleus/cell DNA sequencing data and has been noted by many previous studies \citep{xu2019, szulwach2019, satas2018} (Xu et al 2019 (10.1038/s41598-019-47297-z),  Szulwach et al 2015 (10.1371/journal.pone.0135007), Satas et al 2018 (10.1093/bioinformatics/bty286). To estimate the rate of ADO for each sample we identified heterozygous SNPs called in the snDNA-seq data and evaluated the ratio of heterozygous to homozygous nuclei and determined the rate of ADO to be 8.4\% $\pm$ 4.1\%. We also observed dropout of the constitutional CCM pathogenic allele as evidenced by the presence of WT nuclei in familial CCM samples 5065 and 5073 (Table 1). As a result of ADO, the number of nuclei with all somatic mutations is likely underestimated in each sample. Despite the confounding effects of ADO, all 5 samples clearly indicate that the \italicize{PIK3CA} and CCM gene somatic mutations occur in the same cell.

\subsection{Rapamycin Prevents CCM Formation in Acute Mouse Models}

\section{Discussion}
\subsection{Three-Hit Model of CCM Pathogenesis}
\subsection{Similarities to the Genetic Mechanism of Cancer}
\subsection{Role of Clonal Expansion in Mutagenesis}
\subsection{Therapeutic Implications}
\subsection{Distinct Properties of \italicize{PIK3CA} vs. CCM/\italicize{MAP3K3} Mutations}
\subsection{DVA Predispose to CCM and Other PI3K-Related Diseases}

\section{Methods}
\subsubsection{CCM Collection}
\subsubsection{Brain AVM Collection}
\subsubsection{DNA Extraction}
\subsubsection{Droplet Digital PCR}
\subsubsection{SNaPshot}
\subsubsection{Sequencing}
\subsubsection{Sequence Analysis}
\subsubsection{Single-Nucleus DNA Sequencing}
\subsubsection{Statistics}

\section{Contributions and Acknowledgements}